diff --git a/node_modules/mdast-util-to-hast/lib/all.js b/node_modules/mdast-util-to-hast/lib/all.js
deleted file mode 100644
index ca944e5..0000000
--- a/node_modules/mdast-util-to-hast/lib/all.js
+++ /dev/null
@@ -1,36 +0,0 @@
-'use strict'
-
-module.exports = all
-
-var one = require('./one')
-
-function all(h, parent) {
-  var nodes = parent.children || []
-  var length = nodes.length
-  var values = []
-  var index = -1
-  var result
-  var head
-
-  while (++index < length) {
-    result = one(h, nodes[index], parent)
-
-    if (result) {
-      if (index && nodes[index - 1].type === 'break') {
-        if (result.value) {
-          result.value = result.value.replace(/^\s+/, '')
-        }
-
-        head = result.children && result.children[0]
-
-        if (head && head.value) {
-          head.value = head.value.replace(/^\s+/, '')
-        }
-      }
-
-      values = values.concat(result)
-    }
-  }
-
-  return values
-}
diff --git a/node_modules/mdast-util-to-hast/lib/handlers/blockquote.js b/node_modules/mdast-util-to-hast/lib/handlers/blockquote.js
index e192c93..5917911 100644
--- a/node_modules/mdast-util-to-hast/lib/handlers/blockquote.js
+++ b/node_modules/mdast-util-to-hast/lib/handlers/blockquote.js
@@ -3,7 +3,7 @@
 module.exports = blockquote

 var wrap = require('../wrap')
-var all = require('../all')
+var all = require('../traverse').all

 function blockquote(h, node) {
   return h(node, 'blockquote', wrap(all(h, node), true))
diff --git a/node_modules/mdast-util-to-hast/lib/handlers/delete.js b/node_modules/mdast-util-to-hast/lib/handlers/delete.js
index 518c9e5..771b73d 100644
--- a/node_modules/mdast-util-to-hast/lib/handlers/delete.js
+++ b/node_modules/mdast-util-to-hast/lib/handlers/delete.js
@@ -2,7 +2,7 @@

 module.exports = strikethrough

-var all = require('../all')
+var all = require('../traverse').all

 function strikethrough(h, node) {
   return h(node, 'del', all(h, node))
diff --git a/node_modules/mdast-util-to-hast/lib/handlers/emphasis.js b/node_modules/mdast-util-to-hast/lib/handlers/emphasis.js
index e9114d6..b508f4c 100644
--- a/node_modules/mdast-util-to-hast/lib/handlers/emphasis.js
+++ b/node_modules/mdast-util-to-hast/lib/handlers/emphasis.js
@@ -2,7 +2,7 @@

 module.exports = emphasis

-var all = require('../all')
+var all = require('../traverse').all

 function emphasis(h, node) {
   return h(node, 'em', all(h, node))
diff --git a/node_modules/mdast-util-to-hast/lib/handlers/heading.js b/node_modules/mdast-util-to-hast/lib/handlers/heading.js
index d28ecaa..2e452ce 100644
--- a/node_modules/mdast-util-to-hast/lib/handlers/heading.js
+++ b/node_modules/mdast-util-to-hast/lib/handlers/heading.js
@@ -2,7 +2,7 @@

 module.exports = heading

-var all = require('../all')
+var all = require('../traverse').all

 function heading(h, node) {
   return h(node, 'h' + node.depth, all(h, node))
diff --git a/node_modules/mdast-util-to-hast/lib/handlers/link-reference.js b/node_modules/mdast-util-to-hast/lib/handlers/link-reference.js
index 0c810da..c279170 100644
--- a/node_modules/mdast-util-to-hast/lib/handlers/link-reference.js
+++ b/node_modules/mdast-util-to-hast/lib/handlers/link-reference.js
@@ -4,7 +4,7 @@ module.exports = linkReference

 var normalize = require('mdurl/encode')
 var revert = require('../revert')
-var all = require('../all')
+var all = require('../traverse').all

 function linkReference(h, node) {
   var def = h.definition(node.identifier)
diff --git a/node_modules/mdast-util-to-hast/lib/handlers/link.js b/node_modules/mdast-util-to-hast/lib/handlers/link.js
index 7e1b781..b662219 100644
--- a/node_modules/mdast-util-to-hast/lib/handlers/link.js
+++ b/node_modules/mdast-util-to-hast/lib/handlers/link.js
@@ -1,7 +1,7 @@
 'use strict'

 var normalize = require('mdurl/encode')
-var all = require('../all')
+var all = require('../traverse').all

 module.exports = link

diff --git a/node_modules/mdast-util-to-hast/lib/handlers/list-item.js b/node_modules/mdast-util-to-hast/lib/handlers/list-item.js
index 0d429c1..26bca42 100644
--- a/node_modules/mdast-util-to-hast/lib/handlers/list-item.js
+++ b/node_modules/mdast-util-to-hast/lib/handlers/list-item.js
@@ -3,7 +3,7 @@
 module.exports = listItem

 var u = require('unist-builder')
-var all = require('../all')
+var all = require('../traverse').all

 function listItem(h, node, parent) {
   var result = all(h, node)
diff --git a/node_modules/mdast-util-to-hast/lib/handlers/list.js b/node_modules/mdast-util-to-hast/lib/handlers/list.js
index 8abd4b3..83c5b8a 100644
--- a/node_modules/mdast-util-to-hast/lib/handlers/list.js
+++ b/node_modules/mdast-util-to-hast/lib/handlers/list.js
@@ -3,7 +3,7 @@
 module.exports = list

 var wrap = require('../wrap')
-var all = require('../all')
+var all = require('../traverse').all

 function list(h, node) {
   var props = {}
diff --git a/node_modules/mdast-util-to-hast/lib/handlers/paragraph.js b/node_modules/mdast-util-to-hast/lib/handlers/paragraph.js
index 389e7ab..3586579 100644
--- a/node_modules/mdast-util-to-hast/lib/handlers/paragraph.js
+++ b/node_modules/mdast-util-to-hast/lib/handlers/paragraph.js
@@ -2,7 +2,7 @@

 module.exports = paragraph

-var all = require('../all')
+var all = require('../traverse').all

 function paragraph(h, node) {
   return h(node, 'p', all(h, node))
diff --git a/node_modules/mdast-util-to-hast/lib/handlers/root.js b/node_modules/mdast-util-to-hast/lib/handlers/root.js
index 8d2f5c7..0aa9b15 100644
--- a/node_modules/mdast-util-to-hast/lib/handlers/root.js
+++ b/node_modules/mdast-util-to-hast/lib/handlers/root.js
@@ -4,7 +4,7 @@ module.exports = root

 var u = require('unist-builder')
 var wrap = require('../wrap')
-var all = require('../all')
+var all = require('../traverse').all

 function root(h, node) {
   return h.augment(node, u('root', wrap(all(h, node))))
diff --git a/node_modules/mdast-util-to-hast/lib/handlers/strong.js b/node_modules/mdast-util-to-hast/lib/handlers/strong.js
index fc3e45c..fc63354 100644
--- a/node_modules/mdast-util-to-hast/lib/handlers/strong.js
+++ b/node_modules/mdast-util-to-hast/lib/handlers/strong.js
@@ -2,7 +2,7 @@

 module.exports = strong

-var all = require('../all')
+var all = require('../traverse').all

 function strong(h, node) {
   return h(node, 'strong', all(h, node))
diff --git a/node_modules/mdast-util-to-hast/lib/handlers/table.js b/node_modules/mdast-util-to-hast/lib/handlers/table.js
index d373f1c..f5c4382 100644
--- a/node_modules/mdast-util-to-hast/lib/handlers/table.js
+++ b/node_modules/mdast-util-to-hast/lib/handlers/table.js
@@ -4,7 +4,7 @@ module.exports = table

 var position = require('unist-util-position')
 var wrap = require('../wrap')
-var all = require('../all')
+var all = require('../traverse').all

 function table(h, node) {
   var rows = node.children
diff --git a/node_modules/mdast-util-to-hast/lib/index.js b/node_modules/mdast-util-to-hast/lib/index.js
index b1be3f1..f0895f0 100644
--- a/node_modules/mdast-util-to-hast/lib/index.js
+++ b/node_modules/mdast-util-to-hast/lib/index.js
@@ -7,7 +7,7 @@ var visit = require('unist-util-visit')
 var position = require('unist-util-position')
 var generated = require('unist-util-generated')
 var definitions = require('mdast-util-definitions')
-var one = require('./one')
+var one = require('./traverse').one
 var footer = require('./footer')
 var handlers = require('./handlers')

diff --git a/node_modules/mdast-util-to-hast/lib/revert.js b/node_modules/mdast-util-to-hast/lib/revert.js
index 18489b8..f1006a1 100644
--- a/node_modules/mdast-util-to-hast/lib/revert.js
+++ b/node_modules/mdast-util-to-hast/lib/revert.js
@@ -3,7 +3,7 @@
 module.exports = revert

 var u = require('unist-builder')
-var all = require('./all')
+var all = require('./traverse').all

 // Return the content of a reference without definition as Markdown.
 function revert(h, node) {
diff --git a/node_modules/mdast-util-to-hast/lib/one.js b/node_modules/mdast-util-to-hast/lib/traverse.js
similarity index 63%
rename from node_modules/mdast-util-to-hast/lib/one.js
rename to node_modules/mdast-util-to-hast/lib/traverse.js
index 967ea67..65598e5 100644
--- a/node_modules/mdast-util-to-hast/lib/one.js
+++ b/node_modules/mdast-util-to-hast/lib/traverse.js
@@ -1,9 +1,8 @@
 'use strict'

-module.exports = one
+module.exports = { one, all}

 var u = require('unist-builder')
-var all = require('./all')

 var own = {}.hasOwnProperty

@@ -52,6 +51,37 @@ function text(node) {
   return 'value' in node
 }

+function all(h, parent) {
+  var nodes = parent.children || []
+  var length = nodes.length
+  var values = []
+  var index = -1
+  var result
+  var head
+
+  while (++index < length) {
+    result = one(h, nodes[index], parent)
+
+    if (result) {
+      if (index && nodes[index - 1].type === 'break') {
+        if (result.value) {
+          result.value = result.value.replace(/^\s+/, '')
+        }
+
+        head = result.children && result.children[0]
+
+        if (head && head.value) {
+          head.value = head.value.replace(/^\s+/, '')
+        }
+      }
+
+      values = values.concat(result)
+    }
+  }
+
+  return values
+}
+
 function returnNode(h, node) {
   var clone

